aio/content/marketing/contributors.json

```
@@ -950,6 +950,14 @@
     "website": "https://medium.com/@a.yurich.zuev",
     "bio": "Alexey is a web development addict who likes diving deep into the source code and sharing his knowledge through visualization. Creator of ng-run.com",
     "groups": ["GDE"]
+  },
+  "ainkin": {
```

Please use your GitHub username as key and order alphabetically 🙏 

```suggestion
  "waterplea": {
```